### ISSUE TITLE
"USAGE:" section shows full path

### ISSUE THREAD

**Issue by mkmk88** [codegangsta/cli#299]
When I run my program from the same location the binary is located it shows reasonable "USAGE" (in default help):

```
NAME:
   tftp_demmo - Simple TFTP client

USAGE:
   ./tftp_demo [global options] command [command options] [arguments...]
   
VERSION:
   0.0.0
   
COMMANDS:
   help, h	Shows a list of commands or help for one command
   
GLOBAL OPTIONS:
   --method, -m 	One of following methods: get, put
   --help, -h		show help
   --version, -v	print the version

```
however, if I run the program from other location I see full path in "USAGE" section:

```
NAME:
   tftp_demmo - Simple TFTP client

USAGE:
   ./test1/test2/test4/test3/test5/tftp_demo [global options] command [command options] [arguments...]
   
VERSION:
   0.0.0
   
COMMANDS:
   help, h	Shows a list of commands or help for one command
   
GLOBAL OPTIONS:
   --method, -m 	One of following methods: get, put
   --help, -h		show help
   --version, -v	print the version
```

In my opinion, name of the binary should be stripped and full location shouldn't be visible.

**Comment by TylerPachal** [comment]
I updated my version of the library and now am seeing the same thing.  This may be something that changed recently.

**Comment by ysh7** [comment]
You can use [HelpName](https://github.com/codegangsta/cli/blob/master/app.go#L17) to fix that, though it defaults to os.Args[0] [instead of Name](https://github.com/codegangsta/cli/blob/master/app.go#L74)

**Comment by mkmk88** [comment]
I just set in my application the "HelpName" explicitly:
app.HelpName = "tftp_demo"

**Comment by jszwedko** [comment]
Fixed by #312
